src/views/Map/index.js

```diff
@@ -0,0 +1,210 @@
+
+import React, { Component } from 'react'
+import { Button, Card, CardBody, CardGroup, Col, Container, Form, Input, Label, InputGroupAddon, InputGroupText, Row, FormGroup } from 'reactstrap';
+
+var map, infoWindow, marker, markers;
+
+class CustomerMap extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+
+    }
+    this.googleMapRef = React.createRef()
+    this.initAutocomplete = this.initAutocomplete.bind(this)
+    // this.geoCode = this.geoCode.bind(this)
+  }
+
+
+  componentDidMount() {
+    let googleScript = document.createElement('script');
+    googleScript.src = "https://maps.googleapis.com/maps/api/js?key=xAIzaSyAgD3spwqPBOEayWe0qMEYbkXYg8glLlTY&libraries=places"
+    googleScript.async = true
+    window.document.body.appendChild(googleScript)
+    googleScript.addEventListener('load', this.initAutocomplete)
+  }
+
+  initAutocomplete() {
+    var myLatlng = { lat: 6.60954222424663, lng: 3.370920970651687 };
+    infoWindow = new google.maps.InfoWindow;
+    map = new google.maps.Map(this.googleMapRef.current, {
+      center: myLatlng,
+      zoom: 10,
+      mapTypeId: 'roadmap'
+    });
+
+    // infoWindow = new google.maps.InfoWindow(
+    //   { content: 'Is this your location? <br/> click to select exact', position: myLatlng });
+    // infoWindow.open(map);
+
+
+    map.addListener('click', function (mapsMouseEvent) {
+      // Close the current InfoWindow.
+      infoWindow.close();
+
+      // Create a new InfoWindow.
+      markers.forEach(function (marker) {
+        marker.setMap(null);
+      });
+
+      geoCode(mapsMouseEvent.latLng);
+      console.log(mapsMouseEvent.latLng.toString());
+    });
+
+    if (navigator.geolocation) {
+
+      navigator.geolocation.getCurrentPosition(function (position) {
+        var pos = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        };
+        // marker = new google.maps.Marker({
+        //   position: pos,
+        //   map: map,
+        //   title: 'Your Location!'
+        // });
+
+        // infoWindow.setPosition(pos);
+        // infoWindow.setContent('Is this your location? <br> click on map to change');
+        // infoWindow.open(map, marker);
+
+        markers.forEach(function (marker) {
+          marker.setMap(null);
+        });
+        markers = [];
+        geoCode (pos)
+        // markers.push(marker);
+        console.log(pos.toString());
+        console.log(pos.toString());
+        map.setCenter(pos);
+      }, function () {
+        handleLocationError(true, infoWindow, map.getCenter());
+      });
+
+    } else {
+      // Browser doesn't support Geolocation
+      handleLocationError(false, infoWindow, map.getCenter());
+    }
+    function   handleLocationError (browserHasGeolocation, infoWindow, pos) {
+      infoWindow.setPosition(pos);
+      infoWindow.setContent(browserHasGeolocation ?
+      'Error: The Geolocation service failed.' :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 6.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)